### PR TITLE
[modular] arachne borgs

### PR DESCRIPTION
**Pull request by SodaPoppy**
## About The Pull Request

adds in the arachne borgs from the tg pr https://github.com/tgstation/tgstation/pull/68444

image

it works i just didn't show all of them cause i'm lazy

## How This Contributes To The Skyrat Roleplay Experience

yeah, player choice and all that bullshit

## Changelog

:cl:
add: expanded and resprited arachne borgs
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/altborgs/code/robot_model.dm [hunk 1]
```diff
@@ -205,7 +210,7 @@
 		// PLEASE ASSUME THE POSITION
 		"Protectron" = list(SKIN_ICON = CYBORG_ICON_MED, SKIN_ICON_STATE = "protectron_medical"),
 		"Miss M" = list(SKIN_ICON = CYBORG_ICON_MED, SKIN_ICON_STATE = "missm_med"),
-		"Arachne" = list(SKIN_ICON = CYBORG_ICON_MED, SKIN_ICON_STATE = "arachne"),
+		"Arachne" = list(SKIN_ICON_STATE = "arachne_med", SKIN_ICON = CYBORG_ICON_MED),
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/altborgs/code/robot_model.dm]: This destroys my ocd, please fix

Comment (GoldenAlpharex) [inline modular_skyrat/modules/altborgs/code/robot_model.dm]:
```suggestion
		"Arachne" = list(SKIN_ICON = CYBORG_ICON_MED, SKIN_ICON_STATE = "arachne_med"),
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
It would also be nice to see them all in screenshots, rather just a few.

It is, after all, a requirement for sprite changes to include screenshots of what they look like in-game.

**Comment by SodaPoppy** [comment]
image
image
image

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me!

**Pull request merged**